### ISSUE TITLE
Port rules in Component .hpp file are incorrect 

### ISSUE THREAD

**Issue by martinboers** [PLCnext/PLCnext_CLI#7]
In the `.hpp` file generated by the acfproject template , there are comments that describe the rules for defining GDS ports. At least some of these rules are incorrect.

For example, the comments state:

```text
Create one (and only one) instance of this struct.
Apart from this single struct instance, there must be no other Component variables declared with the #port comment.
```

... but I have successfully broken this rule in firmware version 2021.0.3 on an AXC F 2152.

Perhaps these comments were correct for an earlier firmware version, but they need to be revised.

This might be the same for GDS ports in PLM components (I haven't checked).

**Comment by twilker** [comment]
It is not really a rule. It is more of a suggestion which works. The problem when you use multiple structure is, that it is easy to have the same port name twice. Both in different structures. Do you mean we should change the comment, so that is does not sound like a rule?

**Comment by martinboers** [comment]
These comments are written like rules, and that is good - users of this template need to know what the rules are for defining Component ports, in the same way that users need to know the rules for defining Program ports. After the rules are clear, then examples can be provided. But the first priority is to clearly list the complete set of rules.

So, either the comments should be changed to list the rules for defining Component ports, or else a reference to the complete set of rules should be provided in the comments. Otherwise, I don't understand how users of this template are supposed to know the rules for defining Component ports?

**Comment by twilker** [comment]
Will be fixed with the next version

**Comment by twilker** [comment]
Was fixed in the 21.6 Release.
